content/en/continuous_integration/troubleshooting.md

```diff
@@ -101,6 +101,21 @@ This is done using the following algorithm:
 #### Possible issues with wall time calculation
 If you're using a library for testing time-dependent code, like [timecop][7] for Ruby or [FreezeGun][8] for Python, it is possible that test timestamps are wrong, and therefore calculated wall times. If this is the case, make sure that modifications to time are rolled back before finishing your tests.
 
+### The default branch is not correct
+#### How it impacts the product
+The default branch is used to calculate some of the metrics, namely:
+
+- Wall time: Calculate the wall time percentiles.
+
+- Wall Time comparison for non-default branches: The **vs Default** value is calculated comparing against the default branch.
+
+- New Flaky tests: The known flaky tests are based on the default branch runs, if that is not properly set, it can lead to wrong New Flaky Test detections.
```

Comment:
I think we should be more explicit about what new flaky tests means:
"New flaky tests are tests that were never flaky previously in the default branch: if the default branch is not properly set, this could lead to a wrong number of detected new flaky tests."

Reply:
Rather than explaining what flaky tests are in a troubleshooting section about something different, maybe we should link to the flaky test docs? https://docs.datadoghq.com/continuous_integration/guides/flaky_test_management/

Reply:
You are right, though it seems the explanation on this page is not right https://docs.datadoghq.com/continuous_integration/guides/flaky_test_management/#watch-for-new-flaky-tests